app/models/stolen_record.rb

```diff
@@ -28,6 +28,7 @@ def self.old_attr_accessible
   belongs_to :state
   belongs_to :creation_organization, class_name: "Organization"
   belongs_to :recovering_user, class_name: "User"
+  has_many :theft_alerts, dependent: :destroy
```

Comment:
I would prefer not to `dependent destroy` this, because it's tied to payments (even though it will leave orphan records)